addons/advanced_ballistics/functions/fnc_handleFired.sqf

```diff
@@ -20,7 +20,7 @@
 TRACE_10("firedEH:",_unit, _weapon, _muzzle, _mode, _ammo, _magazine, _projectile, _vehicle, _gunner, _turret);
 
 // Parameterization
-private ["_abort", "_AmmoCacheEntry", "_WeaponCacheEntry", "_opticsName", "_opticType", "_bulletTraceVisible", "_temperature", "_barometricPressure", "_bulletMass", "_bulletLength", "_muzzleVelocity", "_muzzleVelocityShift", "_bulletVelocity", "_bulletLength", "_barrelTwist", "_stabilityFactor", "_aceTimeSecond", "_barrelVelocityShift", "_ammoTemperatureVelocityShift"];
+private ["_abort", "_AmmoCacheEntry", "_WeaponCacheEntry", "_opticsName", "_opticType", "_bulletTraceVisible", "_temperature", "_barometricPressure", "_bulletMass", "_bulletLength", "_muzzleVelocity", "_muzzleVelocityShift", "_bulletVelocity", "_bulletLength", "_barrelTwist", "_stabilityFactor", "_barrelVelocityShift", "_ammoTemperatureVelocityShift"];
```

Comment:
`private` keyword would be faster.

Reply:
?

Reply:
`private _abort = true;`
is evaluated at compile-time.
`private ARRAY` is runtime.
Also private ARRAY assigns nil to that variable. and later when you initialize it you assign again. so using private ARRAY you assign a value to the variable one extra time.

Reply:
We can try to fix that in a separate PR. Feel free to make a Github issue for it.

Reply:
fun-fact. using private keyword is also faster than not using private at all.
using private ARRAY however is slower than not privatizing variables. on average.

Reply:
Yeah.. I'll make a PR that changes it everywhere where appropriate.

Reply:
Sounds good.

Reply:
`private ARRAY` is only in use because some components haven't gotten the love of cleanup yet since `private VAR = VALUE` was introduced.